librosa/effects.py

```diff
@@ -256,8 +259,14 @@ def time_stretch(y, *, rate, **kwargs):
 
 
 def pitch_shift(
-    y, *, sr, n_steps, bins_per_octave=12, res_type="soxr_hq", **kwargs
-):
+    y: np.ndarray,
+    *,
+    sr: float,
+    n_steps: float,
+    bins_per_octave: float = 12,
```

Comment:
bins_per_octave should be integer, not float

Reply:
Done